### PR TITLE
fix: software-vars-init error msgs during prep

### PR DESCRIPTION
**Pull request by rayjh**
Mask missing content logs/errors during --prep

### CODE REVIEW

File: software/wmla120.py [hunk 1]
```diff
@@ -1923,7 +1928,8 @@ def _update_software_vars(self):
                     print(msg)
                     ch, _dir = get_selection(dirs)
                 else:
-                    self.log.error(f'No {repo_id} repo found in software server.')
+                    if self.running != 'prep':
+                        self.log.error(f'No {repo_id} repo found in software server.')
```
Comment (jjalvare) [inline software/wmla120.py]: From a maintenance perspective I suggest you add this to debug and put it in logs. Also, while this software does not have much use from outside users it is always beneficial to keep useful information in logs(errors, cmds, warnings) in the event debug is needed.

File: software/wmla120.py [hunk 2]
```diff
@@ -1671,7 +1674,7 @@ def init_clients(self):
         elif self.sw_vars['ansible_become_pass'] is None:
             cmd += '--ask-become-pass '
             prompt_msg = "\nClient password required for privilege escalation"
-        # Verification Loop
+        # Verfication Loop
```
Comment (jaywcarman) [inline software/wmla120.py]: Typo fixed in PR #517 re-introduced here.

File: software/wmla120.py [hunk 3]
```diff
@@ -1685,10 +1688,10 @@ def init_clients(self):
                 if key not in validation_status:
                     validation_status[key] = f'{self.v_status}'
             print("\n   *** Validation Status ***\n")
-            for key,val in validation_status.items():
+            for key, val in validation_status.items():
                 print(f'{key} = {val}')
 
-            print('\nVerification Completed\n')
+            print('\nVerfication Completed\n')
```
Comment (jaywcarman) [inline software/wmla120.py]: Typo fixed in PR #517 re-introduced here.

File: software/wmla120.py (hunk 1 above)
Comment (rayjh) [reply software/wmla120.py]: done

File: software/wmla120.py (hunk 3 above)
Comment (rayjh) [reply software/wmla120.py]: fixed

File: software/wmla120.py (hunk 2 above)
Comment (rayjh) [reply software/wmla120.py]: fixed

**Pull request merged**